client/src/components/SidebarContainer.jsx

```diff
@@ -37,6 +37,9 @@ const SidebarContainer = (props: Props) => {
 
   const [years, setYears] = useState([]);
   const [arrestData, setArrestData] = useState(null);
+  const [tab, setTab] = useState(0);
```

Comment:
Can we move tab to `MapPage.jsx` and make constants for all values of the tab var (0...3), that way we can update the map when the user switches tabs.